hail/python/hailtop/hailctl/batch/get.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+import yaml
+
+from .batch_cli_utils import get_batch_if_exists
+
+def init_parser(parser):
+    parser.add_argument('batch_id', type=int)
+    parser.add_argument('-o', type=str, default='yaml')
```

Comment:
help/descriptions for these. Also, you can defined possible values for `-o` - right now if I pass "yml" by accident I'll get a type error related to calling a None.